program/src/instruction.rs

```diff
@@ -121,6 +121,9 @@ pub enum PaladinGovernanceInstruction {
         /// The minimum required threshold of proposal rejection to terminate
         /// the proposal.
         proposal_rejection_threshold: u64,
+        /// The Paladin stake config account that this governance config account
+        /// corresponds to.
+        stake_config_address: Pubkey,
```

Comment:
Since we do checks elsewhere that stake accounts and all that are valid and owned by the paladin stake program, we should be consistent, and pass the stake config as an account input to be validated here. That way we protect people from creating unusable governances.

Reply:
We can certainly check that it has proper config state, but we won't be able to do the other checks since we're not providing a stake account here. Want me to add state checks for config?

Reply:
Added. Let me know what you think!

Reply:
Yeah that's the idea, just checking that the config is real, since as you said, there's no stake account